README.md

```diff
@@ -51,19 +52,38 @@ vars:
     lead_history_columns: ['the','list','of','column','names']
```

Comment:
Do we want to make these global and update the name to `marketo__lead_history_columns`?

Reply:
hm not sure -- this variable doesn't get used in the source package, so i feel like it might cause more trouble/breaking changes than it's worth to make it global? open to other opinions tho...